### ISSUE TITLE
More standards-compliant code formats?

### ISSUE THREAD

**Issue by jooh** [esdalmaijer/PyGaze#125]
I just started working on the code, and it's a little tricky to get used to some of the non-standard conventions, especially the super-deep indentation. Would it be worth running the whole repo through [black](https://github.com/ambv/black) to get something more PEP-8 compliant?

Similarly, the windows carriage returns are kind of annoying (top tip - set `git config --local core.whitespace cr-at-eol` to stop them showing up in git diff). Not sure if black will convert file endings anyway, but if not, a pass through dos2unix wouldn't hurt either...

**Comment by esdalmaijer** [comment]
Yeah, we should probably get on 4-space indentation, and a few more PEP8 compliances.

Thanks for the carriage return tips! Most of this stuff is written on a mixture of operating systems, and I don't always pay attention to these things.

**Comment by jooh** [comment]
If you're up for it, I can just run the repo through black and submit a PR. That should get you fairly strict PEP8.
